cypress/integration/pages/liveRadio/testsForCanonicalOnly.js

```diff
@@ -19,7 +20,7 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
 }) =>
   describe(`testsThatFollowSmokeTestConfigForCanonicalOnly for ${service} ${pageType}`, () => {
     beforeEach(() => {
-      cy.getToggles(service);
+      cy.getToggles(config[service].name);
```

Comment:
Can changes in this file be broken out into their own PR? I'm not sure if they relate to recent episodes?

Reply:
Ah - see [my other comment](https://github.com/bbc/simorgh/pull/8597#discussion_r544299981) which relates to using `config[service].name)` instead of just `service`.

Is there a reason for switching?

Reply:
Changed back